.github/workflows/ci.yml

```diff
@@ -4,6 +4,8 @@ on:
   pull_request_target:
     paths-ignore:
       - "**.md"
+      - ".github/workflows/*.yml"
+      - ".github/workflows/*.yaml"
```

Comment:
Why did you add this ignore? Is it because these changes to the workflows won't be run in a `pull_request_target` action anyway?

Reply:
to prevent CI from running in case we're merely adjusting the CI, like in this PR.